org.opent2t.sample.hub.superpopular/com.nest.hub/js/thingTranslator.js

```diff
@@ -182,7 +180,7 @@ class Translator {
      * Gets device details (all fields), response formatted per nest api
      */
     getDeviceDetailsAsync(deviceType, deviceId) {
```

Comment:
Not for this CR. Use "Async" suffix consistently across all async functions

Reply:
Agreed.